### PR TITLE
Update deprecated task module removal in 5.0 documentation

### LINKED ISSUE #6558: no module named celery.task
from celery import shared_task
from .models import Test, Prices

from celery.decorators import periodic_task
from celery.task.schedules import crontab
import requests 
@shared_task
def create_price_items(name):
    Test.objects.create(name=name)
    

@periodic_task(run_every=(crontab(minute='*/1')))
def run_create_items():
    create_price_items.delay(name='new2020')
-------error----------
File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "C:\Users\acer\Desktop\django\open source\dj-blog\src\dj-bitcoin-tracker-api\prices\tasks.py", line 5, in <module>
    **from celery.task.schedules import crontab
ModuleNotFoundError: No module named 'celery.task'**

### PR DESCRIPTION
**Pull request by JensTimmerman**
## Documentation update for removed functionality

Add extra information in documentation and fix outdated documentation concerning the removal of celery.task
Fixes #6558

### CODE REVIEW

**Review by auvipy — COMMENTED** [review]
i'm not sure about shared task only

**Comment by JensTimmerman** [comment]
would you prefer it to say `from celery import shared_task as task` ?

**Review by auvipy — CHANGES_REQUESTED** [review]
shared_task & Task both should work not the small 'task'

File: docs/internals/deprecation.rst [hunk 1]
```diff
@@ -44,10 +44,21 @@ Compat Task Modules
...
     into:
 
+    .. code-block:: python
+
+        from celery import shared_task
+
+    -- and:
     .. code-block:: python
 
         from celery import task
```
Comment (auvipy) [inline docs/internals/deprecation.rst:54]: from celery import task should be Task

File: docs/whatsnew-5.1.rst [hunk 2]
```diff
@@ -290,6 +290,10 @@ you should import `kombu.utils.encoding` instead.
 If you were using the `celery.task` module before, you should import directly
 from the `celery` module instead.
 
+If you were using the `celery.task` decorator you should use 
+`celery.shared_task` instead.
```
Comment (auvipy) [inline docs/whatsnew-5.1.rst:294]: or Task instead

**Review by auvipy — CHANGES_REQUESTED** [review]

File: docs/whatsnew-5.1.rst (hunk 2 above)
Comment (JensTimmerman) [reply docs/whatsnew-5.1.rst:294]: Task is not a decorator, this does not seem to work?

File: docs/internals/deprecation.rst (hunk 1 above)
Comment (JensTimmerman) [reply docs/internals/deprecation.rst:54]: no, shared_task is not a decorator? it is different from shared_task

**Review by auvipy — APPROVED** [review]

File: docs/whatsnew-5.1.rst (hunk 2 above)
Comment (JensTimmerman) [reply docs/whatsnew-5.1.rst:294]: I can add another line for completenes sake: 

```
If you were using `from celery.task import Task` you should use `from celery import Task` instead.
```

Comment (auvipy) [reply docs/whatsnew-5.1.rst:294]: you are right

**Pull request merged**